### PR TITLE
uses zilla slab as iodide logo font on server pages via a google font

### LINKED ISSUE #1028: let's use zilla slab for the "Iodide" at the top of any page
it will match the "moz://a" word mark that we'll want to add to our pages somewhere, and the slab serif "I" will clear up the typographical ambiguity @hamilton has pointed out with "Lodide"

and it's a pretty good looking font too:
image

### PR DESCRIPTION
**Pull request by hamilton**
Closes #1028.

screen shot 2018-10-23 at 10 25 06 pm

### CODE REVIEW

**Comment by wlach** [comment]
@hamilton I probably would, yeah, in case someone wants to deploy iodide in a firewalled environment in the future.

**Comment by hamilton** [comment]
The alternative is to grab the Zilla Slab font from https://github.com/mozilla/zilla-slab and bring in the one style we're using. This would add about 68kb to the bundle, but the whole thing would be portable since you don't need the internet.

**Comment by hamilton** [comment]
@wlach cool - will update this PR accordingly.

File: src/server/style/base.jsx [hunk 1]
```diff
@@ -22,6 +22,12 @@ body {
 /*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
 
 export default injectGlobal`
+ 
+ @font-face {
+  font-family: 'Zilla Slab';
+  src: url("${ZillaSlab}");
+  src: url("/iodide.dev.fonts/ZillaSlab-Light.otf") ;
```
[the remark this answers is not in the document]
Comment (hamilton) [reply src/server/style/base.jsx]: I noticed that this happens on non-index pages primarily, such as `<server>/<username>/`. If I don't include this then it ends up looking at `<server>/<username>/iodide.dev.fonts/ZillaSlab.woff` which doesn't exist. Maybe I should figure out how to get this pathing to work a bit better by making `base.jsx` export a function that calls `injectGlobal`, that way the pathing gets reset for each page.

**Comment by hamilton** [comment]
Going to try compartmentalizing this in the header logo itself so we don't have this fallback issue with `injectGlobal`. This would be less complicated if material-ui wasn't so opinionated with how to handle CSS.

**Comment by hamilton** [comment]
Nevermind - fixed the path, so no need for the hard-coded probably-will-break fallback.

**Pull request merged**